cross-dom-comm/README.md

```diff
@@ -0,0 +1,237 @@
+# How can a contract on one layer control a contract on the other
+
+[![Discord](https://img.shields.io/discord/667044843901681675.svg?color=768AD4&label=discord&logo=https%3A%2F%2Fdiscordapp.com%2Fassets%2F8c9701b98ad4372b58f13fd9f65f966e.svg)](https://discord.com/channels/667044843901681675)
+[![Twitter Follow](https://img.shields.io/twitter/follow/optimismPBC.svg?label=optimismPBC&style=social)](https://twitter.com/optimismPBC)
+
+This tutorial teaches you how to write a contract on one layer (either L1 or L2) that sends a message to call a function on a contract on the other layer. 
```

Comment:
Generally speaking I think it's best if we use "Optimism" and "Ethereum" as opposed to "L1" and "L2". I think that the sort of people most likely to need these tutorials will also be the least familiar with this terminology.